docker/.env.example

```diff
@@ -10,7 +10,7 @@ SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJzZX
 
 ## General
 SITE_URL=http://localhost:3000
-KONG_URL=http://kong:8000
+KONG_URL=http://localhost:8000
```

Comment:
Setting KONG_URL (or supabase-studio's SUPABASE_URL) to localhost breaks user invitation because the studio container then tries to connect to itself:

Failed to invite user: An error has occured: request to http://localhost:8000/auth/v1/invite failed, reason: connect ECONNREFUSED 127.0.0.1:8000

Instead, use:

```suggestion
KONG_URL=http://host.docker.internal:8000
```

Reply:
Then again, this causes:

![image](https://user-images.githubusercontent.com/1864183/153660103-b0518066-6b3e-489a-b9f5-f1a84b2b3c90.png)

I don't see a solution other than:
1. Supabase should specify different URLs, one for backend-to-backend communication and one for frontend-to-backend communication
2. Edit your hosts file and create an alias for 127.0.0.1 other than `localhost`